extensions/ql-vscode/src/remote-queries/variant-analysis-manager.ts

```diff
@@ -73,14 +73,18 @@ export class VariantAnalysisManager extends DisposableObject implements VariantA
   }
 
   public async showView(variantAnalysisId: number): Promise<void> {
-    if (!this.views.has(variantAnalysisId)) {
-      // The view will register itself with the manager, so we don't need to do anything here.
-      this.push(new VariantAnalysisView(this.ctx, variantAnalysisId, this));
+    try{
```

Comment:
I'd personally avoid the big try catch and aim to eliminate errors before calling the functions inside the try/catch. 

The only thing I can think of that would blow up is the existence of variant analysis. I'd check that the variant analysis with the given id exists (`if (!this.variantAnalyses.get(variantAnalysisId)) {...}`)